From f46ba61d7df33d63c1e904c5559017a57a7e8228 Mon Sep 17 00:00:00 2001
From: Stef Walter <stefw@redhat.com>
Date: Fri, 24 Jul 2015 17:03:25 +0200
Subject: [PATCH] tools: Remove spurious print output

---
 tools/realm-join.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/realm-join.c b/tools/realm-join.c
index e5760b2..feb6edc 100644
--- a/tools/realm-join.c
+++ b/tools/realm-join.c
@@ -237,7 +237,6 @@ perform_join (RealmClient *client,
 		return 1;
 	}
 
-g_printerr ("id mapping %d %d\n", args->automatic_id_mapping_set, args->automatic_id_mapping);
 	options = realm_build_options (REALM_DBUS_OPTION_COMPUTER_OU, args->computer_ou,
 	                               REALM_DBUS_OPTION_MEMBERSHIP_SOFTWARE, args->membership_software,
 	                               REALM_DBUS_OPTION_USER_PRINCIPAL, args->user_principal,
-- 
2.4.3

